dev/sg/internal/release/release.go

```diff
@@ -188,6 +193,66 @@ var Command = &cli.Command{
 	},
 }
 
+// Return type from releaseregistry
+// for the /releases/sourcegraph endpoint
+type releaseInfo struct {
+	ID         int    `json:"id"`
+	Name       string `json:"name"`
+	Public     bool   `json:"public"`
+	CreatedAt  string `json:"created_at"`
+	PromotedAt string `json:"promoted_at"`
+	Version    string `json:"version"`
+	GitSha     string `json:"git_sha"`
+}
+
+// Determines latest major.minor.patch version number by hitting the releaseregistry
```

Comment:
nit: godoc conventions are to start the comment with the function name. 

```suggestion
// determineNextReleaseVersion determines latest major.minor.patch version number by hitting the releaseregistry
```